app/Helpers/Payment/EventPayment.php

```diff
@@ -52,9 +52,14 @@ public function getPackagePrice(): int
         return $this->package === null ? 0 : $this->package->price;
     }
 
+    public function getDiscount(): float
```

Comment:
Misschien deze `getDiscountFactor()` noemen? Iedere keer bij het lezen denk ik: stel ik krijg 30% korting, is deze discount factor dan 0.3 of 0.7 :')  maar ik weet niet echt duidelijkere naamgeving hiervoor

Reply:
Ja ik had dit ook, ik dacht dat discount factor beter was. Want normaal druk je discount uit als 30% dus 0.3 . Dus ik  dacht, discount factor is dan 0.7 want dat is de vermenigvuldiging factor.

Reply:
renamed